src/style.css

```diff
@@ -1,3 +1,46 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 100px;
+  width: 1024px;
+  top: 37px;
+  left: -601px;
```

Comment:
The 'width' and 'left' properties of the '.header' class seem to be hardcoded, which might not be responsive or suitable for different screen sizes. Consider using relative units or media queries to make the layout more flexible.